javascript-light-bulb/main.js

```diff
@@ -0,0 +1,14 @@
+var count = 0;
+var $lightbulb = document.querySelector('.lightbulb');
+var $background = document.querySelector('.background');
+$lightbulb.addEventListener('click', increment);
+function increment() {
+  count++;
+  if (count % 2 === 0) {
```

Comment:
Rather than using a counter, I would use a boolean to keep track of if the light is on or off 🤓